src/CopyrighFixer/CopyrighFixer/configparser.h

```diff
@@ -0,0 +1,34 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#ifndef CONFIGPARSER_H
+#define CONFIGPARSER_H
+
+#include "CopyrighFixer_global.h"
+#include "config.h"
+
+namespace CopyrighFixer {
+
+/**
+ * @brief The ConfigParser class that parse the settings received at the input of the program.
+ */
+class ConfigParser {
+public:
+    ConfigParser();
+
+    /**
+     * @brief parseOptions This a method that will parse all input options.
+     * @return The config object.
```

Comment:
* add more description about object, because it is result value, objects that set in this argument will be changed after invoke this method.
* Your function return bool if the method finished successful but you don not describe it.